chore: fix EditorConfig lint errors (issue #6213)

AlyAbdelmoneim opened the pull request:
Resolves #6213.

## Description

> What is the purpose of this pull request?

This pull request:

- Fixes indentation issues in `defaults.json` files by replacing tabs with spaces to comply with EditorConfig rules.

## Related Issues

> Does this pull request have any related issues?

This pull request:

- Resolves #6213 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

AlyAbdelmoneim committed:
Fix EditorConfig indentation: replace tabs with spaces in defaults.json

AlyAbdelmoneim committed:
style: reformat indentation to 2 spaces

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

AlyAbdelmoneim commented:
@aayush0325 Hello, thanks for the approval, it seems that I didn't have the signature correct on the commits so the merging was blocked, I fixed that, does it need another approval ?

kgryte commented on lib/node_modules/@stdlib/random/streams/betaprime/lib/defaults.json:
```diff
@@ -1,7 +1,7 @@
-	"sep": "\n",
-	"copy": true,
-	"siter": 1e308
+    "objectMode": false,
```
These files should be two-space indentation. Currently, they are 4-space indentation.

kgryte replied:
Would you mind going ahead and updating the two changed files accordingly?

AlyAbdelmoneim replied:
sure

AlyAbdelmoneim committed:
chore: resolve merge conflict

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

AlyAbdelmoneim commented:
@kgryte done !
